demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -226,6 +229,10 @@ def detect_file_level(file_path: str) -> PathLevel:
         if dir_name in CONTENT_ENTITIES_DIRS:
             return PathLevel.CONTENT_ENTITY_DIR
 
+        path_without_last_dir = file_path.rstrip(f'{dir_name}').rstrip('/')
```

Comment:
might be wiser to use:
`str(os.path.dirname(path)).endswith(GENERIC_TYPES_DIR)`

This way you can support windows devs as well.